Nav BridgeIT/css/style.css

```diff
@@ -0,0 +1,92 @@
+*{
+  box-sizing: border-box;
+  margin: 0px;
+  padding: 0px;
+}
+
+body{
+  font-size: 62.5%;
+}
+
+.flexWrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-items: center;
+  font-size: 1.5rem;
+  width: 100%;
+  position: absolute;
```

Comment:
1. Projekt zakłada, że menu będzie "przyklejone" do okna przeglądarki dlatego w tym wypadku nelażałoby dać właściwość fixed.
2. Nawet jeżeli projekt by nie zakładał "przyklejenia" to pozycja absolutna jest i tak zbędna, ponieważ menu i tak będzie pierwszym elementem w drzewie.